client_opts.go

```diff
@@ -185,6 +186,12 @@ func WithPullLabels(labels map[string]string) RemoteOpt {
 	}
 }
 
+// WithContainerLayerIntegrity enables integrity protection of WCOW/LCOW layers by
+// setting appropriate RemoteContext label
+func WithContainerLayerIntegrity() RemoteOpt {
```

Comment:
Makes me wonder whether we should have this option, I don't see any RemoteOpt's before this that only work for a specific platform. I don't think supplying the `WithPullLabel` bit would be too cumbersome and would leave all of the options freely usable on *nix and Windows

Reply:
yeah, makes sense. maybe I'll just limit the change to our fork.